frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx

```diff
@@ -47,20 +39,17 @@ const ConfiguratePodcastForm: React.FC<Props> = ({
   return (
     <form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
       <fieldset disabled={isFormDisabled} className={styles.fieldset}>
-        <Input
-          name={PodcastPayloadKey.NAME}
+        <ImagePreviewControl
+          name={PodcastPayloadKey.IMAGE}
```

Comment:
hmm, do we need to pass an existing image? 🤔

Reply:
> hmm, do we need to pass an existing image? 🤔

I think so.
If I don't need it, may you tell me why?